### ISSUE TITLE
Support external media requests

### ISSUE THREAD

**Issue by turt2live** [bifurcation/ietf-mimi-protocol#24]
Where we assume the external attachment blob ("media") is stored on a follower server, another server/the hub needs a way to fetch that media on behalf of its local clients.

We do not want to support clients requesting the media directly.

TODO: Discuss authentication/right to download.

**Comment by rohanmahy** [comment]
An option articulated in #75

**Comment by tgeoghegan** [comment]
Making progress here is blocked on writing down a proper threat model, tracked in #93.

**Comment by rohanmahy** [comment]
See now #111

**Comment by rohanmahy** [comment]
PR #111  merged into mimi-protocol and need for threat model is in #93. Closing.
